module3/labs/Week4/QueryStringParameters.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Query String Parameters Lab
+---
+
+### Background
+The luxurious mountain ski hotel company *Summit Slope Retreats* has contracted a team of developers to build an app that will allow customers to view their many incredible hotels.
+
+You've just joined the team working on this project, and today's your first day!
+
+The team has already started building the application. Start by cloning [this](https://github.com/turingschool-examples/MountainHotels) repo.
+
+### Task 1: Getting The App Up and Running
+Your first task is to get the application up and running and see at least two test properties. This task is a bit more tricky than it might originally seem. It's not as easy as just pressing run! Once you have the app up and running, move on to Task 2, your instructor will call everyone back together shortly to go over Task 1.

```

Comment:
Can we change this to `see at least two test hotels`? 

It might be good to indicate that when they are set up, they should see something similar to the image below when they visit the hotels index.